src/style.css

```
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf") format("truetype"),
+    url("/fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-weight: 300;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.search-input-small,
+.search-input,
+.search-form {
+  border: none;
+  margin: 0;
+  padding: 0;
+  background-color: transparent;
+}
```

```suggestion
.search-input,
.search-form {
  border: none;
  margin: 0;
  padding: 0;
  background-color: transparent;
}
```
and do you really need this?